contrib_bots/lib/DefineBot/DefineBot.py

```diff
@@ -0,0 +1,85 @@
+# See readme.md for instructions on running this code.
+import json
+import requests
+
+class DefineHandler(object):
+    '''
+    This plugin facilitates creating define tasks when
```

Comment:
This documentation string is incorrect. It should be updated to reflect the actual function of this bot, including that it defines words (not creates tasks); and that it responds in the same stream (not in a new stream).